src/coreclr/src/System.Private.CoreLib/src/System/Internal.cs

```diff
@@ -13,6 +13,7 @@
 using System;
```

Comment:
given this file is identical between mono and coreclr, would you mind moving it to https://github.com/dotnet/runtime/tree/master/src/libraries/System.Private.CoreLib/src/Internal so we can share it?

maybe rename it to a more descriptive name like `AssemblyAttributes.cs` or something like that.

Reply:
done!